tests/docker/Dockerfile

```diff
@@ -72,22 +72,25 @@ RUN apt update && \
     echo 'root soft nofile 65535' >> /etc/security/limits.conf && \
     echo 'root hard nofile 65535' >> /etc/security/limits.conf
 
+# Retry options for all following curl calls
+ENV CURL_FLAGS="--retry 3 --retry-connrefused --retry-delay 2"
```

Comment:
There is an `ENV` block further up, but I'm deliberately setting this further down so that when we change it, it doesn't trigger a slow rebuild of the Java parts.